Apps/Playground/CMakeLists.txt

```diff
@@ -49,11 +64,12 @@ if(APPLE)
         set_source_files_properties(${SCRIPTS} ${BABYLON_SCRIPTS} ${DEPENDENCIES} PROPERTIES MACOSX_PACKAGE_LOCATION "Resources/Scripts")
         set_source_files_properties(${REFERENCE_IMAGES} PROPERTIES MACOSX_PACKAGE_LOCATION "Resources/ReferenceImages")
     endif()
-    set(ADDITIONAL_LIBRARIES ${ADDITIONAL_LIBRARIES}
-        PRIVATE ${JAVASCRIPTCORE_LIBRARY}
-        PRIVATE NativeCamera)
+    set(ADDITIONAL_LIBRARIES ${ADDITIONAL_LIBRARIES} PRIVATE ${JAVASCRIPTCORE_LIBRARY})
     set(RESOURCE_FILES ${STORYBOARD})
 
+    # Remove -Wall and -Werror from the debug flags to make simulators work.
```

Comment:
Do you have some infos on errors/ warnings here ?

Reply:
Yes, https://github.com/BabylonJS/BabylonNative/pull/1384#discussion_r1727289518